tests/queries/0_stateless/03141_wildcard_grants.sql

```diff
@@ -20,5 +20,6 @@ SHOW GRANTS FOR user_03141;
 SELECT '---';
```

Comment:
`nit:` Should we also have a test case for when the database name empty? `... ON .* TO ...`